src/HealthGPS.Console/program.cpp

```diff
@@ -112,8 +112,23 @@ int main(int argc, char *argv[]) { // NOLINT(bugprone-exception-escape)
 #ifdef CATCH_EXCEPTIONS
     try {
 #endif
+        // In future, we want users to supply the data source via the config file only, but for now
+        // we also allow passing it via a command line argument. Sanity check: Make sure they only
+        // do one of these things!
+        if (!(cmd_args.data_source.has_value() ^ config.data_source.has_value())) {
```

Comment:
I guess this could just be `if (A == B)`. But the XOR operator is cooler I suppose.

Reply:
Oh yeah :facepalm:. Good spot. I've done this somewhere else so I'll fix them both